bn-apps/memberships-management/membership-service/src/test/kotlin/net/corda/businessnetworks/membership/RevokeMembershipFlowTest.kt

```diff
@@ -49,7 +50,30 @@ class RevokeMembershipFlowTest : AbstractFlowTest(2) {
         assert(notifiedParties.toSet() == participantsNodes.map { identity(it) }.toSet())
     }
 
+    @Test
```

Comment:
The same as above. Should we extract it to a separate test?

Reply:
yeah, it's kinda the same flow I thought just different entrypoint to it but can do

Reply:
i think it would be better to keep this in the same test class given it all relates to activating memberships. Then we don't end up with that many test classes, which makes it hard to orientate in.